### ISSUE TITLE
Sample request: Hosting and activity dependency injection

### ISSUE THREAD

**Issue by jakejscott** [temporalio/samples-dotnet#29]
### Describe the solution you'd like

Sample showing how to use the Temporalio.Extensions.Hosting package

**Comment by cretz** [comment]
I was planning on doing this after I merge #28 which is waiting on release which is waiting on a NuGet bug

**Comment by jakejscott** [comment]
Duplicate of https://github.com/temporalio/samples-dotnet/issues/9
